modules/installation-configuration-parameters.adoc

```diff
@@ -2165,12 +2156,6 @@ ifeval::["{context}" == "installing-openstack-user-sr-iov-kuryr"]
 :!osp:
 :!osp-kuryr:
 endif::[]
-ifeval::["{context}" == "installing-vsphere-installer-provisioned-customizations"]
-:!vsphere:
-endif::[]
-ifeval::["{context}" == "installing-vsphere-installer-provisioned-network-customizations"]
-:!vsphere:
-endif::[]
```

Comment:
Worth noting that while this work is for IBM Power VS, I am now just noticing that https://github.com/openshift/openshift-docs/pull/60438, inadvertently adding these directives back to the module. These directives are no longer used and where originally removed with https://github.com/openshift/openshift-docs/pull/61165